kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -60,12 +60,21 @@ If you copy the sample scripts to a different location, make sure that you copy
 The default domain created by the script has the following characteristics:
 
 * An Administration Server named `admin-server` listening on port `7001`.
-* A dynamic cluster named `cluster-1` of size 2.
+* A dynamic cluster named `cluster-1` of size 5.
 * Two Managed Servers, named `managed-server1` and `managed-server2`, listening on port `8001`.
 * No applications deployed.
-* A Derby data source.
 * A T3 channel.
 
+If you run the sample from a machine that is remote to the Kubernetes cluster, and you need to push the new image to a registry that is local to the cluster, you need to do the following:
+* Set the `image` property in the inputs file to the target image name (including the tag if needed).
+* Run the `create-domain.sh` script without the `-e` option.
+* Push the generated image `domain-home-in-image-wdt:lastest` (or `domain-home-in-image:latest`) to the registry.
```

Comment:
If set the image property in the inputs file, the property value should be the generated image name. Maybe we need change the statement as following:
Push the generated image `image` (or `domain-home-in-image-wdt:lastest`, or `domain-home-in-image:latest`) to the registry.

site/quickstart.md also need same change.